NameGame/NameGame/APIService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  APIService.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+/**
+    Reads and models data from online API.
+ */
+
+class APIService {
+    
+    static let manager = APIService()
```

Comment:
We will want to move away from a share instance in the future, but for now let's change the name to `shared`.